### ISSUE TITLE
Chef 12.18 installs JSON 2.0.2, causing a conflict with right_api_client and it's JSON 1.x dependency

### ISSUE THREAD

**Issue by rgeyer** [rightscale-cookbooks/machine_tag#34]
Chef 12.18.31 specifically instals JSON 2.0.2, which causes a conflict when running rs-base, which depends upon machine_tag.

This creates an unresolvable gem dependency map.

```
[0mGem::ConflictError[0m
------------------[0m
Unable to activate right_api_client-1.6.2, because json-2.0.2 conflicts with json (~> 1.0)[0m

[0mCookbook Trace:[0m
---------------[0m
/var/chef/cache/cookbooks/machine_tag/libraries/machine_tag_rl10.rb:30:in `initialize_api_client'
[0m/var/chef/cache/cookbooks/machine_tag/libraries/machine_tag_rl10.rb:35:in `api_client'
[0m/var/chef/cache/cookbooks/machine_tag/libraries/machine_tag_rl10.rb:43:in `create'
[0m/var/chef/cache/cookbooks/machine_tag/libraries/provider_machine_tag.rb:44:in `action_create'[0m

[0mResource Declaration:[0m
---------------------[0m
# In /var/chef/cache/cookbooks/rightscale_tag/recipes/default.rb
[0m
[0m 28:   machine_tag "server:uuid=#{node['rightscale']['instance_uuid']}"
[0m 29: end
[0m
[0mCompiled Resource:[0m
------------------[0m
# Declared in /var/chef/cache/cookbooks/rightscale_tag/recipes/default.rb:28:in `from_file'
[0m
[0mmachine_tag("server:uuid=03-E5U1B0D371IFO") do
[0m  provider Chef::Provider::MachineTag
[0m  action :create
[0m  retries 0
[0m  retry_delay 2
[0m  default_guard_interpreter :default
[0m  declared_type :machine_tag
[0m  cookbook_name "rightscale_tag"
[0m  recipe_name "default"
[0mend
[0m
[0mPlatform:[0m
---------[0m
x86_64-linux[0m
```

**Comment by rshade** [comment]
this is hopefully fixed in #35

**Comment by rshade** [comment]
Completed
